posthog/models/filters/test/test_retention_filter.py

```diff
@@ -35,11 +56,39 @@ def test_fill_date_from_and_date_to(self):
         self.assertEqual(
             filter.to_dict(),
             {
-                "date_to": "2020-08-02T00:00:00+00:00",
+                "date_to": "2020-08-01",
```

Comment:
@EDsCODE after merging in your "major filter refactoring" I had to change this. Does this still look correct?

Reply:
yeah I added a comment to describe the difference. i've made it so to_dict will only provide values that have been set which means _date_to and _date_from on the filter object. Before, we were returning derived/default values that date_from/date_to set if there isn't an underlying _date_from/_date_to. The retention filter adds a period to the derived value which is why there's that 1 day difference